services/movie_session.py

```diff
@@ -0,0 +1,44 @@
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: int,

```

Comment:
The parameter `movie_show_time` should have a type hint of `datetime` instead of `int` to correctly represent the show time.